robottelo/vm.py

```diff
@@ -87,6 +87,8 @@ def __init__(
         self.ip_addr = None
         self._created = False
         self._target_image = str(id(self))
+        if tag is not None:
+            self._target_image = tag + self._target_image
```

Comment:
If a user passes in a non-string value as tag, isn't that a PEBKAC issue? This code looks fine to me.